pandas/core/indexing.py

```diff
@@ -2371,7 +2371,7 @@ def ravel(i):
 
             # we have a frame, with multiple indexers on both axes; and a
             # series, so need to broadcast (see GH5206)
-            if sum_aligners == self.ndim and all(is_sequence(_) for _ in indexer):
+            if all(is_sequence(_) or isinstance(_, slice) for _ in indexer):
```

Comment:
L2379 below uses `len(indexer[1])`. Won't this fail if `indexer[1]` is a slice?

Reply:
You are right. There was a problem with the slice. Fixed it.